helm/prometheus-rules/templates/grafana-agent-rules.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.mimir.enabled }}
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+    {{- include "labels.common" . | nindent 4 }}
+  name: grafana-agent-rules
+  namespace: {{ .Release.Namespace | quote }}
+spec:
+  catalog: giantswarm
+  config:
+    configMap:
+      name: grafana-agent-rules-config
+      namespace: {{ .Release.Namespace | quote }}
+    secret:
+      name: ""
+      namespace: ""
+  kubeConfig:
+    context:
+      name: ""
+    inCluster: true
+    secret:
+      name: ""
+      namespace: ""
+  name: grafana-agent
+  namespace: {{ .Values.namespace | quote }}
```

Comment:
I couldn't set `mimir` as namespace, I got that error:
```
Reason:         Upgrade "prometheus-rules" failed: failed to create resource: admission webhook "apps.app-admission-controller.giantswarm.io" denied the request: validation error: target namespace mimir is not allowed for in-cluster apps
```

Reply:
That's thé one at the top that Iz causing thé issue L9 should bé giantswarm

Reply:
It's the case with `.Release.Namespace` no ?

Reply:
`monitoring`